src/test/java/org/opensearch/sdk/TestSDKClusterService.java

```diff
@@ -110,20 +109,16 @@ public void testAddSettingsUpdateConsumer() throws Exception {
 
         // Before initialization should store pending update but do nothing
         sdkClusterService.getClusterSettings().addSettingsUpdateConsumer(boolSetting, boolConsumer);
-        verify(extensionsRunner, times(0)).getExtensionTransportService();
```

Comment:
Why are all these `verify()` calls being removed? They are an integral part of the test.